### DISCUSSION TITLE
Some thoughts regarding version v1.0.977

### DISCUSSION THREAD

**Discussion by ElTopo** [lollipopkit/flutter_server_box#411] [Ideas]
1. A little bit wider for the percentage number area in CPU usage section, so "100" won't split into 2 lines:
![cpu](https://github.com/lollipopkit/flutter_server_box/assets/217195/af9dc7aa-5bb3-4c12-87c3-a3df02941e0f)

2. In Disk section, can '/' (i.e., the root of all file systems) be always displayed even it's not a regularl disk type, or can I pick fs items from` df -h`? (for example, my / has overlay type for overlay rootfs)

the first `tmpfs `item in the screenshot also looks weird and I don'tk now why it appeared here:

![disk](https://github.com/lollipopkit/flutter_server_box/assets/217195/b6f83067-e154-49ff-af9e-30ad94fe2317)

3. It would be nice if mobile_v##.sh files are saved temporarily in memory (such as in /dev/shm), instead of permanently on user's persistent storage (~/.config/server_box)

**Comment by lollipopkit** [comment]
Great! Track it at #443
